test/constructor.js

```diff
@@ -32,11 +31,11 @@ describe('@constructor@', function () {
 
   it("should launch nemo with @noConfigPath@overrideArg@", function (done) {
     delete process.env.nemoBaseDir;
-    nemo = Nemo({
+    Nemo({
       "driver": {
         "browser": "phantomjs"
       }
-    }, function () {
+  }, function (err, nemo) {
```

Comment:
All of your spacings for where you do this are de-indented one chunk from where they should be.